app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):

```

Comment:
u don't need to inherit from ABC one more time, u already did it because Player is inherited from ABC